deepinv/tests/test_physics.py

```diff
@@ -1054,27 +1054,32 @@ def test_tomography(device):
         for parallel_computation in [True, False]:
             for fan_beam in [True, False]:
                 for circle in [True, False]:
-                    imsize = (1, 16, 16)
-                    physics = dinv.physics.Tomography(
-                        img_width=imsize[-1],
-                        angles=imsize[-1],
-                        device=device,
-                        circle=circle,
-                        fan_beam=fan_beam,
-                        normalize=normalize,
-                        parallel_computation=parallel_computation,
-                    )
-
-                    x = torch.randn(imsize, device=device).unsqueeze(0)
-                    r = (
-                        physics.A_adjoint(physics.A(x))
-                        * PI.item()
-                        / (2 * len(physics.radon.theta))
-                    )
-                    y = physics.A(r)
-                    error = (physics.A_dagger(y) - r).flatten().mean().abs()
-                    epsilon = 0.2 if device == "cpu" else 0.3  # Relax a bit of GPU
-                    assert error < epsilon
+                    for adjoint_via_backprop in [True, False]:
```

Comment:
One further formatting comment, please can you pull from #541 and adapt your contributions to the test similarly? i.e. use pytest parametrize rather than for loop. Thanks!